projects/angular/src/accordion/accordion-panel.ts

```diff
@@ -42,6 +45,12 @@ export class ClrAccordionPanel implements OnInit, OnChanges {
   @Input('clrAccordionPanelDisabled') @HostBinding('class.clr-accordion-panel-disabled') disabled = false;
   @Input('clrAccordionPanelOpen') panelOpen = false;
 
+  @Input('clrAccordionPanelHeading') heading = false;
+  /**
+   * Level of the accordion/stepper heading from 1 to 7.
+   */
+  @Input('clrAccordionPanelHeadingLevel') _explicitHeadingLevel: HeadingLevel;
```

Comment:
```suggestion
  @Input('clrAccordionPanelHeadingLevel') explicitHeadingLevel: HeadingLevel;
```

I don't think a underscore is needed here. That's usually only used for fields that back a property of the same same. That's not really the case here. I think "explicit" replaces the needed for the underscore while also making the code more understandable.

Reply:
Corrected in commit: 37c1f1b